sdks/python/run_pylint.sh

```diff
@@ -72,6 +72,7 @@ ISORT_EXCLUDED=(
   "iobase_test.py"
   "fast_coders_test.py"
   "slow_coders_test.py"
+  "executor.py"
```

Comment:
<!--thread_id:cc_150367422_t; commit:59f2c74a04bd13869ca4998935cf14f2d8c3a74f; resolved:0-->
<!--section:context-quote-->
> **aaltay** wrote:
> What is the problem with isort in this file?

<!--section:body-->
It was complaining about the wrong import order for some reason. I have tried again without and it works without excluding the file.